tensorboard/components/tf_ng_tensorboard/reducers.ts

```diff
@@ -37,7 +37,7 @@ export function logger(reducer: ActionReducer<any>): ActionReducer<any> {
 }
 
 // TODO(stephanwlee): Create dev mode and conditionally enable this.
-export const metaReducers: MetaReducer<any>[] = false ? [logger] : [];
+export const metaReducers: MetaReducer<any>[] = true ? [logger] : [];
```

Comment:
Yes. Because the code is in active development and is not part of production build anyways, I decided to flip this one. Ideally, we need something like node-env-inline which we lack.